src/script/util/DOM/resizeObserver.ts

```diff
@@ -46,7 +46,8 @@ export const useResizeObserver = (
   callback: (element: Element | HTMLDivElement) => void,
   element?: Element | HTMLDivElement | null,
 ) => {
-  useEffect(() => {
+  // We need to use a layout effect here as we want to make sure the observer is set up (and removed!) before the component is rendered
+  useLayoutEffect(() => {
```

Comment:
And this is the reason why the initial PR (https://github.com/wireapp/wire-webapp/pull/16846) was not enough